modules/01 JavaScript Programming/Assignments/20/answer.js

```diff
@@ -0,0 +1,20 @@
+var arr = [1, 2, 5, 7];
+
+let myFilter = function(someArr, callbackFunc) {
+     if (typeof callbackFunc === 'function') return callbackFunc(someArr);
+     else console.log('ERROR: myFilter function must get a function as a second param');
+}
+
+let getEvenArrNums = function(someArr) {
+     var filteredArr = [];
+
+     someArr.forEach( num => {
+          if (num % 2 === 0) filteredArr.push(num);
+     });
+
+     return filteredArr;
+}
+
+var filtered = myFilter(arr, getEvenArrNums);
+
+console.log(filtered);
```

Comment:
Correct. Your answer and Shimon's are the same. Please make sure that both of you understand what the code does